npm-postinstall.js

```diff
@@ -2,23 +2,39 @@ const fs = require('fs');
 const path = require('path');
 const cp = require('child_process');
 
-const submodules = [
-  'project',
-  'modules/electrophysiology_browser/jsx/react-series-data-viewer',
-];
-
-submodules.forEach((submodule) => {
-  // Check if the submodule is in modules/, and has an override
-  const submodulePath = submodule.split(path.sep);
+/**
+ * Check if the path provided is a module and has an override
+ * in project. If so, returns the override path
+ * @param {string} p
+ * @return {string}
+ */
+const getPath = (p) => {
+  const pathParts = p.split(path.sep);
 
   if (
-    submodulePath[0] === 'modules' &&
-    fs.existsSync(path.join(__dirname, 'project', 'modules', submodulePath[1]))
+    pathParts[0] === 'modules' &&
+    fs.existsSync(path.join(__dirname, 'project', 'modules', pathParts[1]))
   ) {
-    submodule = path.join('project', submodule);
+    return path.join('project', p);
   }
 
-  submodule = path.join(__dirname, submodule);
+  return p;
+};
+
+/*
+ * ----------------------------------------
+ * Install npm packages in LORIS submodules
+ * ----------------------------------------
+ */
+const eegVisualization =
+  'modules/electrophysiology_browser/jsx/react-series-data-viewer';
+const submodules = [
+  'project',
```

Comment:
I believe this array is not structured as intended. This is looking for the file 'project/package.json' on line 40.

Reply:
What is the issue?

Reply:
After discussing on Slack I understand why this is needed.

No change is necessary. This works as intended.